native-image-workshop/README.md

```diff
@@ -136,25 +145,33 @@ We can also run the `native-image` tool using Maven. If you look at the `pom.xml
 ```xml
 <!-- Native Image -->
 <plugin>
-    <groupId>org.graalvm.nativeimage</groupId>
-    <artifactId>native-image-maven-plugin</artifactId>
-    <version>${graalvm.version}</version>
+    <groupId>org.graalvm.buildtools</groupId>
+    <artifactId>native-maven-plugin</artifactId>
+    <version>${native.maven.plugin.version}</version>
+    <extensions>true</extensions>
     <executions>
-        <execution>
-            <goals>
-                <goal>native-image</goal>
-            </goals>
-            <phase>package</phase>
-        </execution>
+    <execution>
+        <id>build-native</id>
+        <goals>
+        <goal>build</goal>
+        </goals>
+        <phase>package</phase>
+    </execution>
     </executions>
     <configuration>
         <!-- Set this to true if you need to switch this off -->
         <skip>false</skip>
         <!-- The output name for the executable -->
         <imageName>${exe.file.name}</imageName>
-        <!-- Set any parameters you need to pass to native-image tool.-->
+        <mainClass>${app.main.class}</mainClass>
         <buildArgs>
-            --no-fallback --no-server
+            <!-- Don't create a antive exe that falls back to starting a JVM if the image won't build -->
```

Comment:
... a antive exe  > a native executable that falls back to starting a JVM if the image doesn't build

Reply:
updated